packages/components/pages/f-mfa/src/tenants/en-GB.js

```diff
@@ -1,4 +1,32 @@
-export default {
+const messages = {
     locale: 'en-GB',
-    text: 'I am a VMfa Component (GB)'
+    text: 'I am a VMfa Component (GB)',
+    verificationPage: {
+        subTitle: "We've emailed your login code",
+        instructionsPrimaryText: "We've sent a verification code to <b>{email}</b> to help keep your account secure. Enter it below to log in.",
```

Comment:
I think this should be `<strong>{email}</strong>`.

Reply:
Will update this soon, at the same time as replacing the `v-html`